README.md

```diff
@@ -1 +1,143 @@
+# hrxp-api
 
+API and backend for the HRX Project.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Database Tables](#database-tables)
+  - [Schema](#schema)
+- [Endpoints](#endpoints)
+- [Data requests and responses](#Data-requests-and-responses)
+
+# Getting Started
+
+- Run `npm i` from the command line.
```

Comment:
Can you add a description of what these two CLI do?

Reply:
Valid point!